.github/workflows/e2e-android.yml

```diff
@@ -0,0 +1,142 @@
+name: e2e-android
+
+on:
+  pull_request:
+
+jobs:
+  e2e:
+    runs-on: macos-10.15
+    timeout-minutes: 30
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # -- Setup --
+    - name: Setup Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14
+
+    - name: Setup Java
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'adopt'
+        java-version: '8'
```

Comment:
This is because of the Detox android setup guide that explicitly mentions it as required [here](https://github.com/wix/Detox/blob/master/docs/Introduction.AndroidDevEnv.md#java-setup)